### PR TITLE
Deliverable 4 david

### PR DESCRIPTION
**Pull request by 7thStringofZhef**
I wrote comments for all my tests and wrote up and tested an email sender. Still have work to do, but I figure it'd be good to have a commented up version of code on the master branch.

Also just added my file receiving function.

### CODE REVIEW

File: skeleton-web-services/src/main/resources/static/js/main.js [hunk 1]
```diff
@@ -5913,6 +5890,10 @@ var m = require('mithril'),
     LoginComponent = require('./components/Login/Login'),
     ProfileComponent = require('./components/Profile/Profile');
 
+function verify() {
+    //
```
Comment (AdityaMahadevan) [inline skeleton-web-services/src/main/resources/static/js/main.js]: verify() either needs to be implemented or removed?

File: skeleton-web-services/src/test/java/cs1530/osgfreesurfer/services/JobServiceTest.java [hunk 2]
```diff
@@ -36,6 +36,7 @@ public void tearDown()
 	}
 	
 	@Test
+	//Test service's find all function. There should be the initial 2 jobs
```
Comment (AdityaMahadevan) [inline skeleton-web-services/src/test/java/cs1530/osgfreesurfer/services/JobServiceTest.java]: The comments fo all of these tests look good :+1:

**Comment by 7thStringofZhef** [comment]
Hmm...that's the frontend chunk I loaded in. Let me try pulling the newest frontend set from github and trying it again.

File: skeleton-web-services/target/classes/static/js/main.js [hunk 3]
```diff
@@ -5913,6 +5890,10 @@ var m = require('mithril'),
     LoginComponent = require('./components/Login/Login'),
     ProfileComponent = require('./components/Profile/Profile');
 
+function verify() {
+    //
```
Comment (AdityaMahadevan) [inline skeleton-web-services/target/classes/static/js/main.js]: another empty verify() ?

File: skeleton-web-services/src/main/java/cs1530/osgfreesurfer/services/UserServiceBean.java [hunk 4]
```diff
@@ -85,4 +90,15 @@ public boolean checkIfLoggedOn(long id) {
 		return loggedOn;
 	}
 
+	@Override
+	public void emailUser(String email)
```
Comment (AdityaMahadevan) [inline skeleton-web-services/src/main/java/cs1530/osgfreesurfer/services/UserServiceBean.java]: :+1: (assuming it works)

**Comment by 7thStringofZhef** [comment]
I found a problem with email and fixed it. I also added session support, but that forces people to install and run Redis, so I also left instructions on how to disable that function. 

I pulled the newest version of the frontend down and packaged it in, but if something's wrong with verify() or w/e, there's really nothing I can do about it. It's not my function, and I don't know the first thing about JS

**Comment by AdityaMahadevan** [comment]
Looks like the new version of the frontend doesn't even have that file with the verify function anymore. Also, it sounds like using something like Redis might need approval? 

We can discuss during the next sprint. I'm going to merge this now.

**Pull request merged**